howler/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.8.1" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "2.0.5")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/howler
+       :version     +version+
+       :description "Javascript audio library for the modern web."
+       :url         "https://howlerjs.com/"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "https://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url (format "https://unpkg.com/howler@%s/dist/howler.js" +lib-version+)
+             :checksum "ac45047bc5f0b0b97aa22fd52bb29c5a"
```

Comment:
No need to use `download :checksum` with `validate-checksums` task.

And there probably should be a `boot-cljsjs-checksums.edn` on the dir after running the package task, you should commit that to git.